AppActionVisualizer/app/src/main/java/com/example/appactionvisualizer/utils/Utils.java

```diff
@@ -98,59 +101,95 @@ private static Toast getToast(String message, Context context, int length) {
     return Toast.makeText(context, message, length);
   }
 
-  //go to a web page with url
+  // go to a web page with url
   public static void jumpToWebPage(Context context, final String url) {
     context.startActivity(new Intent(Intent.ACTION_VIEW, Uri.parse(url)));
   }
 
-  //first check if package exists since user may haven't installed the app
-  //then check if the url can jump to activity
+  // first check if package exists since user may haven't installed the app
+  // then check if the url can jump to activity
   public static void jumpToApp(Context context, final String url, final String packageName) {
     try {
       context.getPackageManager().getApplicationInfo(packageName, 0);
     } catch (PackageManager.NameNotFoundException e) {
       Utils.jumpToWebPage(context, context.getString(R.string.url_reference_prefix, packageName));
     }
     try {
-      //parseUri() might throw URISyntaxException error
+      // parseUri() might throw URISyntaxException error
       Intent intent = Intent.parseUri(url, 0);
       intent.setPackage(packageName);
-      //startActivity() might throw ActivityNotFoundException error
+      // startActivity() might throw ActivityNotFoundException error
       context.startActivity(intent);
     } catch (Exception e) {
```

Comment:
catch(URISyntaxException e){
....
      }
 catch(ActivityNotFoundException e){
....
      }
  catch(Exception e){
      }

Reply:
Looks much better in this way, will fix